src/java.desktop/share/classes/sun/java2d/cmm/lcms/LCMS.java

```diff
@@ -53,7 +53,7 @@ public Profile loadProfile(byte[] data) {
         return null;
     }
 
-    private static LCMSProfile getLcmsProfile(Profile p) {
+    static LCMSProfile getLcmsProfile(Profile p) {
         if (p instanceof LCMSProfile) {
             return (LCMSProfile)p;
         }
```

Comment:
Would be a good time to
```suggestion
        if (p instanceof LCMSProfile profile) {
            return profile;
        }
```

Reply:
That change is possible but it will be better to do it as a separate cleanup.